### ISSUE TITLE
Organise submissions by topic not date

### ISSUE THREAD

**Issue by BackSpaceTech** [aws/aws-week-in-review#216]
Having the week in review organised by date makes it very difficult to find the wood from the trees. Also creates lumps of submissions for Monday.
Can you please consider having submissions ordered by topic? The topics could be organised like the AWS Blog (Architecture, DevOps, PHP Development, .NET Development, Ruby Development, Mobile Development, Java Development, Security, Startup, Big Data, Database Blog, Partner Network, Compute, AWS for SAP, SES, Internet of Things, Public Sector). 
This could also be used to allocate review of submissions to the appropriate AWS department.

**Comment by ebekker** [comment]
I agree it would make it more easy to digest the announcements by topic, but it is also less *systematic* than listing them out by date.  The latter is indisputable and stated as fact, where as attributing a topic could become very subjective and open to interpretation.  What if an entry falls into multiple topics?  What if it doesn't fall into any existing topic?

**Comment by hyandell** [comment]
I discussed with @jeffbarr. The Week in Review is no longer being published. You can visit https://aws.amazon.com/new/ to track the latest product, service, and feature announcements from AWS.
